SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLoadBalancingHandler.cpp

```diff
@@ -152,3 +152,62 @@ uint64 FSpatialLoadBalancingHandler::GetLatestAuthorityChangeFromHierarchy(const
 
 	return LatestTimestamp;
 }
+
+void FSpatialLoadBalancingHandler::LogMigrationFailure(EActorMigrationResult ActorMigrationResult, AActor* Actor)
+{
+	FString FailureReason;
+
+	if (Actor->GetGameTimeSinceCreation() > 1)
+	{
+		switch (ActorMigrationResult)
+		{
+		case EActorMigrationResult::NotAuthoritative:
+			FailureReason = TEXT("does not have authority");
+			break;
+		case EActorMigrationResult::NotReady:
+			FailureReason = TEXT("is not ready");
+			break;
+		case EActorMigrationResult::PendingKill:
+			FailureReason = TEXT("is pending kill");
+			break;
+		case EActorMigrationResult::NotInitialized:
+			FailureReason = TEXT("is not initialized");
+			break;
+		case EActorMigrationResult::Streaming:
+			FailureReason = TEXT("is streaming in or out");
+			break;
+		case EActorMigrationResult::NetDormant:
+			FailureReason = TEXT("is startup actor and initially net dormant");
+			break;
+		case EActorMigrationResult::NoSpatialClassFlags:
+			FailureReason = TEXT("does not have spatial class flags");
+			break;
+		case EActorMigrationResult::DormantOnConnection:
+			FailureReason = TEXT("is dormant on connection");
+			break;
+		default:
+			FailureReason = TEXT("");
+			break;
+		}
+	}
+	else
```

Comment:
Assigning to TEST("") is not necessary, just testing FailureReason.IsEmpty() should be enough.

Reply:
Sure, done.